docs/data-sources/aws_assume_role_policy.md

```diff
@@ -46,6 +46,7 @@ resource "databricks_mws_credentials" "this" {
 ## Argument Reference
 
 * `external_id` (Required) (String) External ID that can be found at http://accounts.cloud.databricks.com/#aws
+* `for_log_delivery` (Optional) Either or not this assume role policy should be created for usage log delivery. Defaults to false.
```

Comment:
nit: if its a bool, probably name it as is_log_delivery?

Reply:
i was thinking about it, for does it make sense semantically? it's not common in tf to have `is_` prefixes. nor for our apis...
`log_delivery_principal = true` was the other idea